content/post/115-protocol-labs-and-igalia-collaborating-on-dweb-in-browsers.md

```diff
@@ -0,0 +1,194 @@
+* [2610100: Add tests for SecurityOrigin::IsPotentiallyTrustworthy](https://chromium-review.googlesource.com/c/chromium/src/+/2610100)
+* [2612947: Remove SecurityPolicy APIs for handling a trustworthy safelist](https://chromium-review.googlesource.com/c/chromium/src/+/2612947)
+* [2614784: Remove SchemeRegistry APIs for handling local and secure schemes](https://chromium-review.googlesource.com/c/chromium/src/+/2614784)
+* [2615260: Implement SecurityOrigin::IsPotentiallyTrustworthy with network::IsOriginPotentiallyTrustworthy](https://chromium-review.googlesource.com/c/chromium/src/+/2615260)
+* [2617709: Rewrite SecurityOrigin::IsSecure() using GURL and url::Origin](https://chromium-review.googlesource.com/c/chromium/src/+/2617709)
+* [2617883: Make SecurityOrigin::IsSecure treat localhost and local files as secure](https://chromium-review.googlesource.com/c/chromium/src/+/2617883)
+</details>
+
+<details><summary>Firefox</summary>
+
+* [Improve compatibility of protocol_handlers with registerProtocolHandler](https://phabricator.services.mozilla.com/D71747)
+* [Safelist cabal, dat, did, dweb, ethereum, hyper, ipfs, ipns, and ssb schemes for registerProtocolHandler().](https://phabricator.services.mozilla.com/D71580)
+* [Remove the title argument from registerProtocolHandler()](https://phabricator.services.mozilla.com/D71714)
+* [ Do not set network.dns.ipv4OnlyDomains when running XPCShell](https://phabricator.services.mozilla.com/D94986)
+* [Add a test to ensure loopback host names cannot be overridden](https://phabricator.services.mozilla.com/D94712)
+* [Hardcode localhost to loopback](https://phabricator.services.mozilla.com/D92716)
+
+</details>
+
+<details><summary>WebKit</summary>
+
+* [Don't treat loopback addresses (127.0.0.0/8, ::1/128, localhost, .localhost) as mixed content](https://bugs.webkit.org/show_bug.cgi?id=171934)
+* [Don't treat loopback IP addresses (127.0.0.0/8, ::1/128) as mixed content](https://bugs.webkit.org/show_bug.cgi?id=218623)
+* [Introduce preference not to treat localhost and .localhost as mixed content](https://bugs.webkit.org/show_bug.cgi?id=218627)
+* [[GTK] Allow WebKitTestServer to run non-loopback addresses for API test](https://bugs.webkit.org/show_bug.cgi?id=219257)
+* [Migrate WebKitTestServer to libsoup 2.48 API](https://bugs.webkit.org/show_bug.cgi?id=219160)
+* [Treat loopback addresses (127.0.0.0/8, ::1/128, localhost, .localhost) as potentially trustworthy URL ](https://bugs.webkit.org/show_bug.cgi?id=218980)
+
+</details>
+
+## Conclusion
+
+Protocol Labs and Igalia have made initial effort this year to improve support and interoperability of web platform features that would benefit the distributed web as well as the web community in general. In addition to start discussions among the different actors, several patches have already landed in browsers. We are looking forward to continuing this next year... Stay tuned!
```

Comment:
I'm not sure this is "wrong" but I'd suggest that either "an initial effort" or "have made initial efforts" would read more clearly imo...